app/src/main/java/com/example/qrranger/MainActivity.java

```diff
@@ -1,14 +0,0 @@
-package com.example.qrranger;
```

Comment:
This is not up to date with the `main` branch. The `MainActivity.java` file in main is populated. See this: 

https://github.com/CMPUT301W23T48/QR-Ranger/blob/main/app/src/main/java/com/example/qrranger/MainActivity.java#L1

I believe you should merge main into this branch first and then we can merge this PR.

Same thing for `activity_main.xml`